client/Challenges.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+
+class Challenges extends Component {
+  render() {
+    // fake some data to render
+    const ChallengesToRender = [
+      { name: 'challenge1', description: 'draw a circle', difficulty: 1 },
+      { name: 'challenge2', description: 'draw a square', difficulty: 1 },
+    ];
+    return (
+      <div>
+        <ul>
+          {ChallengesToRender.map((challenge, idx) => {
+            return <li key={challenge.name}>{`${challenge.name}: ${challenge.description}`}</li>;
```

Comment:
Fine for now... we need to set up routes that connect to these links and have our "IndividualChallenge" page pull in the route parameter.